### DISCUSSION TITLE
mark above as read

### DISCUSSION THREAD

**Discussion by Fl0rent** [FreshRSS/FreshRSS#5616] [Q&A]
When you have a long list of articles to read, and little time, it would be nice to be able to mark the top articles as read.
The Reeder application has this function, and I find it indispensable. And one day I'd like to be able to read my feeds directly on my freshrss instance and not on my mobile.
I can't trust the function that marks articles as read when you scroll.

Maybe there's a plugin but I haven't found it.
![IMG_BA61087AA52F-1](https://github.com/FreshRSS/FreshRSS/assets/1076038/22840f17-b396-4b3e-93bb-58c4714b6918)

**Comment by Alkarex** [comment]
Hello,
We might indeed introduce a menu to perform several actions on a given article.

For the time being though, you can already use the the *mark as read while scrolling option*:

![image](https://github.com/FreshRSS/FreshRSS/assets/1008324/3170ff81-1b62-4a6a-8d56-e9a7fc2cf0f8)

Another possibility is that several of the mobile apps supporting FreshRSS have this feature:
https://github.com/FreshRSS/FreshRSS/blob/edge/README.md#apis--native-apps

**Reply by Fl0rent** [reply]
yes thanks for your answer !
I already use the the mark as read while scrolling option, but I don't find that it works the way I want it to. Basically, sometimes I know I've scrolled over articles, and yet when I come back they're still there. I need a visual cue to see which articles are being read in real time.

Yes, I use reeder for that, and it's perfect.
